### DISCUSSION TITLE
Support continue open-source autopilot

### DISCUSSION THREAD

**Discussion by d1y** [zed-industries/zed#26778] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

https://github.com/continuedev/continue

![image.png](https://s2.loli.net/2024/02/18/ERmLwCObMj93P1o.png)

**Comment by iamnbutler** [comment]
Hi – a link is not an issue – could you describe the feature you are envisioning further?

**Comment by dhruvkelawala** [comment]
If I can provide some context, **continue** has 2 major use cases:

- Select the code and ask AI to edit it. Something like "Optimise the code".
- Select the code and add it to the Chat conversation to ask questions about it.

There are some terminal features as well for debugging etc. 

Adding this feature/extension by default in Zed will really be a game changer!
